Services/Common/OJS.Services.Common.Data/Implementations/SubmissionsForProcessingCommonDataService.cs

```diff
@@ -80,6 +80,8 @@ public async Task AddOrUpdateBySubmissionIds(ICollection<int> submissionIds)
             .Select(sId => new SubmissionForProcessing
             {
                 SubmissionId = sId,
+                Processed = false,
+                Processing = false,
             });
 
         using var scope = TransactionsHelper.CreateTransactionScope();
```

Comment:
Leaving a comment just so i can request changes as i've found a bug in local testing